src/Carousel.svelte

```diff
@@ -15,6 +16,14 @@
       (currentIndex =
         currentIndex != 0 ? currentIndex - 1 : loop ? items.length - 1 : 0),
   };
+  export let autoplayInterval = -1; // autoplay disactivated by default
+  const autoplay = () => {
+    loop = true; // sets loop to true when autoplaying
+    setInterval(controls.next, autoplayInterval);
+  };
+  onMount(() => {
+    autoplayInterval > 0 ? autoplay() : {};
```

Comment:
the interval should be cleared on destroy to prevent memory leaks.

Reply:
Indeed, just pushed a fix on that